craft.lic

```diff
@@ -429,7 +429,7 @@ class Craft
 
     wait_for_script_to_complete('sew', ['trash', 'knitting', chapter, unique_name, item, 'wool'])
 
-    bput('stow my yarn', 'You put', 'Stow what')
+    bput("put my yarn in my #{@bag}", 'You put', 'What were')
```

Comment:
This may be a good time to refactor them to `DRCI. put_away_item?('yarn', @bag)`